src/App.js

```diff
@@ -25,7 +26,7 @@ const App = () => {
   return (
     <div className="App">
       <Route exact path='/' render={() => <Login setUser={setUser}/>} />
-      <Route exact path='/home' render={() => <Home {...homeProps}/>} />
+      <Route exact path='/home' render={() => projects ? <Home {...homeProps}/> : <Loader />} />
```

Comment:
I couldn't see the loader in action, even when commenting out the `setProjects` on line 18. 
We may need to change the conditional on line 29 to `projects.length ? <Home.......`

Before we merge this PR, Could you test this on your end?  Changing `projects` to `projects.length` on my end seemed to work.

Reply:
@hfaerber No problem, I added another commit. I was also able to verify that it's working on my end.